app/javascript/stylesheets/osbbs.scss

```diff
@@ -29,10 +29,6 @@
   align-items: center;
 }
 
-div a.btn-outline-primary {
-    margin-left: 30px;
-  }
-
```

Comment:
Doesn't that change the styles for all the buttons? If so, make sure that these changes do not affect other pages, if not required

Reply:
I checked it, everything is okay